### PR TITLE
Challange exit script

### PR DESCRIPTION
**Pull request by eezcjkr**
connects https://github.com/leapdao/meta/issues/55

Exit script + integ test

### CODE REVIEW

File: tests/1_challangeExit.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  let state = await node.getState();
+  console.log(state);
+  let aliceUnspent = unspentForAddress(state.unspent, alice, color).map(u => ({
+    output: u.output,
+    outpoint: Outpoint.fromRaw(u.outpoint),
+  }));
+  let inputs = [new Input(aliceUnspent[0].outpoint)];
+  let outputs = [new Output(amount, alice.toLowerCase(), color)];
+  let firstTransfer = Tx.transfer(inputs, outputs).signAll(alicePriv);
```
Comment (troggy) [inline tests/1_challangeExit.js]: can be replaced with `Tx.transferFromUtxos` ?

File: tests/1_challangeExit.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  console.log(exit);
+
+  await challenge(firstTransfer.hash(), 
+    secondTransfer.hash(), 
+    node.web3.currentProvider.host, 
+    web3.currentProvider.host, 
+    challengerPriv);
+
+  exit = await contracts.exitHandler.methods.exits(utxoId).call();
+  console.log(exit);
```
Comment (troggy) [inline tests/1_challangeExit.js]: should we check here that `challenge` removed the fraudlent exit from the queue?

File: tests/1_challangeExit.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+const { Tx, Input, Output, Outpoint, Period, helpers } = require('leap-core');
```
Comment (troggy) [inline tests/1_challangeExit.js]: typo in the filename `1_challangeExit.js` → `1_challengeExit.js`

**Comment by eshavkun** [comment]
Didn't find any assertions. Do you plan on adding them?

File: tests/1_challangeExit.js (hunk 1 above)
Comment (eezcjkr) [reply tests/1_challangeExit.js]: Okay

File: tests/1_challangeExit.js (hunk 2 above)
Comment (eezcjkr) [reply tests/1_challangeExit.js]: Yeah was about to do that.

File: tests/1_challangeExit.js (hunk 3 above)
Comment (eezcjkr) [reply tests/1_challangeExit.js]: Cool

**Comment by eezcjkr** [comment]
Yeah so the integration tests are broken due to BigInt stuff. And also the tests are kind off out of the scope of this, we just a need a script that works as stand-alone. I mainly used this repo to help me develop the script. But would be nice to somehow fit it into existing tests also.

File: README.md [hunk 4]
```diff
@@ -62,3 +62,9 @@ accounts = [{addr, privKey}, {addr, privKey}...]
 // web3 is just Web3 contected to the ganache network
 web3 = Web3
 ```
+
+## Challange exit script
```
Comment (troggy) [inline README.md]:
```suggestion
## Challenge exit script
```

**Comment by pinkiebell** [comment]
continued @ https://github.com/leapdao/leap-guardian/pull/1

**Pull request closed without merging**